### ISSUE TITLE
false positive [E1101(no-member)] Module 'scipy.special' has no 'erfinv' member

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#3744]
### Steps to reproduce
```python
import scipy.special as sp
print(sp.erfinv(0))
```

### Current behavior

```
z.py:2: [E1101(no-member), ] Module 'scipy.special' has no 'erfinv' member
```

### Expected behavior

no diagnostics

### pylint --version output
```
pylint 2.5.3
astroid 2.4.2
Python 3.8.4 (default, Jul 14 2020, 02:58:48)
[Clang 11.0.3 (clang-1103.0.32.62)]
```

**Comment by AWhetter** [comment]
I can't reproduce this, even with the same versions of pylint and astroid. What version of scipy are you using?

**Comment by sam-s** [comment]
```
>>> scipy.__version__
'1.5.2'
```

**Comment by hippo91** [comment]
@sam-s thanks for the report.
I can reproduce with master branches of `pylint` and `astroid`, `scipy` 1.19.1 and `python` 3.7.7.
`scipy.special` should need an `astroid` brain to be handled correctfully.
Otherwise you may try to whitelist the module in your configuration.

**Comment by sam-s** [comment]
Thanks, I use
```
z = sp.erfinv(p) * np.sqrt(2)  # https://github.com/PyCQA/pylint/issues/3744 pylint: disable=no-member
```
instead.

**Comment by sam-s** [comment]
```
pylint 3.3.8
astroid 3.3.11
Python 3.13.3 (main, Aug 14 2025, 11:53:40) [GCC 14.2.0]
```

`pyproject.toml`:
```
[tool.pylint.'MESSAGES CONTROL']
extension-pkg-allow-list = [
    "scipy.special",            # https://github.com/PyCQA/pylint/issues/3744
]
```
does not help

**Comment by sam-s** [comment]
I use too many [`scipy.special`](https://docs.scipy.org/doc/scipy/reference/special.html) functions to mark them all, and [whitelisting](https://github.com/pylint-dev/pylint/issues/3744#issuecomment-3315476383) does not work, even with
```
pylint 4.0.2
astroid 4.0.1
Python 3.13.3 (main, Aug 14 2025, 11:53:40) [GCC 14.2.0]
```
